pkg/provider/acme/local_store.go

```diff
@@ -31,10 +31,16 @@ func NewLocalStore(filename string) *LocalStore {
 
 func (s *LocalStore) save(resolverName string, storedData *StoredData) {
 	s.lock.Lock()
```

Comment:
To follow up my first comment.

```suggestion
	s.lock.Lock()
	defer s.lock.Unlock()
```